.github/workflows/publish_to_hex_pm.yml

```
@@ -33,11 +29,26 @@ jobs:
         with:
           ref: ${{ github.event.workflow_run.head_branch || github.ref }}
 
-      - name: Set up Elixir ${{ env.ELIXIR_VERSION }} and OTP ${{ env.OTP_VERSION }}
-        uses: erlef/setup-elixir@v1
+      - name: Setup asdf
+        uses: asdf-vm/actions/setup@v1
+
+      - name: Cache asdf
+        uses: actions/cache@v2
         with:
-          otp-version: ${{ env.OTP_VERSION }}
-          elixir-version: ${{ env.ELIXIR_VERSION }}
+          path: |
+            /home/runner/.asdf
```

Since you just use 1 path, should we make it inline?
```suggestion
          path: /home/runner/.asdf
```